src/i18n/RuleEngine.js

```diff
@@ -1265,4 +1265,8 @@ It's recommended to use a template syntax, e.g., \`\${'{'}timestamp{'}'}\` or \`
     zh: '输入 JSON 以生成 JSON Schema',
     en: 'Enter JSON to generate JSON Schema',
   },
+  JSONSchemaVersionTip: {
+    zh: `当前支持 JSON Schema draft-03、draft-04 和 draft-06 版本。`,
+    en: `JSON Schema versions draft-03, draft-04, and draft-06 are supported.`,
```

Comment:
maybe we can highlight the `draft-xxx` with markdown format.

Reply:
sorry.. I just saw it.